registries/_format/base64url.md

```diff
@@ -0,0 +1,21 @@
+---
+owner: baywet
+issue:
+description: A url-safe binary array as defined in RFC4648
```

Comment:
Is this being added purely for OAS 3.0?  In OAS 3.1 this should be `"contentEncoding": "base64url"`.

Reply:
I guess the format registry shows yup with the latest spec so we should design this documentation with 3.1 in mind. That being said OpenApi.net doesn't support 3.1 just yet (we're working on it) and kiota as of today looks at things from a 3.0 perspective to provide some context.
Do you think we should add a note here saying that with 3.1 people should refrain from using this and use the content encoding instead?

Reply:
Yes, because the point of aligning OAS 3.1 and JSON Schema 2020-12 was to enable using off-the-shelf JSON Schema 2020-12 tools as much as possible. And those should know `"contentEncoding": "base64url"` but likely will not know `"format": "base64url"` (if they even pay attention to `format` at all, which many tools do not).

Reply:
noted, waiting for a resolution on the binary discussion before I push a change.

Reply:
added